litr/src/main/java/com/linkedin/android/litr/Annotations.kt

```diff
@@ -0,0 +1,9 @@
+package com.linkedin.android.litr
+
+@Retention(value = AnnotationRetention.BINARY)
+@RequiresOptIn(
+    level = RequiresOptIn.Level.WARNING,
+    message = "The frame extract APIs are experimental in LiTr, and may be changed or removed in the future."
+)
+@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION, AnnotationTarget.TYPEALIAS, AnnotationTarget.PROPERTY)
+annotation class ExperimentalFrameExtractorApi
```

Comment:
This is very nifty way to mark experimental APIs. Let's generalize this, by renaming it to '@ExperimentalApi`

Reply:
Rationale for making experimental annotation specific to a feature was that, if we add a new experimental feature, then consumers might unknowingly be opting into it (e.g. when someone opts into `@OptIn(LiTrExperimentalApi::class)`, and we add a new feature `FooExperiment`, they won't get any warnings)